mocks/openedx/core/djangoapps/plugins/constants.py

```diff
@@ -0,0 +1,11 @@
+"""
+Mocks for openedx.core.djangoapps.plugins.constants.
+
+This mostly stupid mocks just to make the plugin app works.
+"""
+
+from mock import Mock
+
+# Just a lazy way to avoid complex mocking.
```

Comment:
Another way to look at it is that this provides a minimal mocking to reduce test maintenance costs

Reply:
It's always better to see the filled half of the glass.